pkg/cache/placeholder_test.go

```diff
@@ -130,37 +142,6 @@ func TestNewPlaceholder(t *testing.T) {
 	assert.Equal(t, "", holder.pod.Spec.PriorityClassName)
 }
 
-func TestNewPlaceholderWithLabelsAndAnnotations(t *testing.T) {
```

Comment:
Move extra assertions in TestNewPlaceholderWithLabelsAndAnnotations() to TestNewPlaceholder()